step-5/README.md

```diff
@@ -74,12 +81,14 @@ shuffleCards() {
   });
 },
 ```
-Wow! Your original `shuffleCards()` function from the `script.js` file was about 12 lines of code, and had a _time cost_ to compute and change DOM element values.
+
+Wow! Your original `shuffleCards()` function from the `script.js` file was about 12 lines of code, and had a [_time cost_](https://web.dev/why-speed-matters/#performance-is-about-user-experience) to compute and change DOM element values.
 Your VueJS version is 9 lines of code and will execute MUCH faster. (You may not notice the speed difference, since as we learned previously, JavaScript can be very FAST on its own)
 
-To see this in action, call the `shuffleCards()` method inside a `mounted() {}` property. `mounted()` is a Vue Lifecycle Hook. It will only run whenever the current component (`GameBoard.vue`) mounts to the DOM. This is similar to a page refresh, except that it is a _component_ refresh.
+To see this in action, call the `shuffleCards()` method inside a [`mounted()` property](https://vuejs.org/api/options-lifecycle.html#mounted). `mounted()` is a [Vue Lifecycle Hook](https://vuejs.org/guide/essentials/lifecycle.html). It will only run whenever the current component (`GameBoard.vue`) mounts to the DOM. This is similar to a page refresh, except that it is a _component_ refresh.
```

Comment:
this one is tough - I like the docs for Vue Lifecycle Hooks on the vuejs.org site because the diagram is a great visual. However I'm not a fan of the intense programming jargon at https://vuejs.org/api/options-lifecycle.html#mounted to explain the mounted hook. This particular documentation is really about the `mounted` hook in the VueJS 3.0 Composition API, which I think is too advanced for this assignment. I applaud you for finding that link in the VueJS docs, because clearly the basic guidelines don't come with a **basic** explanation of what a mounted hook does 😞

What do you think of this resource:
https://www.digitalocean.com/community/tutorials/vuejs-component-lifecycle#understanding-mounting-hooks-dom-insertion
for explaining the `mounted` hook?

I realize I cannot link directly to the mounted hook there, but the explanation found there might be easier to understand for newcomers. Do you agree?

Reply:
I do agree. I have updated to the DigitalOcean link